File: Integraupt-Backend/BackendAdReserva/src/main/java/com/AdReserva/dto/AdminReservaSummaryDto.java
package com.AdReserva.dto;

public record AdminReservaSummaryDto(
        long pendientes,
        long aprobadas,
        long rechazadas,
        long canceladas
) {
    public static AdminReservaSummaryDto empty() {
        return new AdminReservaSummaryDto(0, 0, 0, 0);
    }
}

File: Integraupt-Backend/servicio-adreserva/src/DTO/AdminReservaSummaryDto.php
<?php

namespace App\DTO;

class AdminReservaSummaryDto {
    public int $pendientes;
    public int $aprobadas;
    public int $rechazadas;
    public int $canceladas;

    public function __construct(int $pendientes, int $aprobadas, int $rechazadas, int $canceladas) {
        $this->pendientes = $pendientes;
        $this->aprobadas = $aprobadas;
        $this->rechazadas = $rechazadas;
        $this->canceladas = $canceladas;
    }

    public static function empty(): self {
        return new self(0, 0, 0, 0);
    }
}
